compiler/src/dotty/tools/dotc/ast/Desugar.scala

```diff
@@ -520,7 +520,14 @@ object desugar {
           enumCases.last.pushAttachment(DesugarEnums.DefinesEnumLookupMethods, ())
         val enumCompanionRef = TermRefTree()
         val enumImport =
-          Import(enumCompanionRef, enumCases.flatMap(caseIds).map(ImportSelector(_)))
+          Import(enumCompanionRef, enumCases.flatMap(caseIds).map{
```

Comment:
No need for braces. I would write 
```scala
...map(enumCase =>
  ...
)
```

Reply:
Changed, it got much more simplified thanks to the other comments.